components/ca-import.js

```diff
@@ -296,7 +290,7 @@ define([
                     })
                     .then(data => {
                         trace('CSV file converted and posted');
-                        this.onsave.call(this, data);
+                        this.onsave.call(data);
```

Comment:
ooopps, sorry. If I'm being honest, I wasn't paying much attention....trying to defeat the Lynel on Zelda while refactoring yields no fruits in either case.